Specs/React.podspec

```diff
@@ -0,0 +1,212 @@
+                                the best product development framework available,
+                                with an emphasis on iteration speed, developer
+                                delight, continuity of technology, and absolutely
+                                beautiful and fast products with no compromises in
+                                quality or capability.
+                             DESC
+  s.homepage                = "http://facebook.github.io/react-native/"
+  s.license                 = package["license"]
+  s.author                  = "Facebook"
+  s.source                  = source
+  s.default_subspec         = "Core"
+  s.requires_arc            = true
+  s.platforms               = { :ios => "8.0", :tvos => "9.2" }
+  s.pod_target_xcconfig     = { "CLANG_CXX_LANGUAGE_STANDARD" => "c++14" }
+  s.preserve_paths          = "package.json", "LICENSE", "LICENSE-docs", "PATENTS"
+  s.cocoapods_version       = ">= 1.2.0"
+
+  s.subspec "Core" do |ss|
+    ss.dependency             "yoga", "#{package["version"]}.React"
+    ss.source_files         = "React/**/*.{c,h,m,mm,S,cpp}"
+    ss.exclude_files        = "**/__tests__/*",
+                              "IntegrationTests/*",
+                              "React/DevSupport/*",
+                              "React/Inspector/*",
+                              "ReactCommon/yoga/*",
+                              "React/Cxx*/*",
+    ss.ios.exclude_files    = "React/**/RCTTV*.*"
+    ss.tvos.exclude_files   = "React/Modules/RCTClipboard*",
+                              "React/Views/RCTDatePicker*",
+                              "React/Views/RCTPicker*",
+                              "React/Views/RCTRefreshControl*",
+                              "React/Views/RCTSlider*",
+                              "React/Views/RCTSwitch*",
+                              "React/Views/RCTWebView*"
+    ss.header_dir           = "React"
+    ss.framework            = "JavaScriptCore"
+    ss.libraries            = "stdc++"
+    ss.pod_target_xcconfig  = { "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\"" }
+  end
+
+  s.subspec "CxxBridge" do |ss|
+    ss.dependency             "Folly", "2016.09.26.00"
+    ss.dependency             "React/Core"
+    ss.dependency             "React/cxxreact"
+    ss.compiler_flags       = folly_compiler_flags
+    ss.private_header_files = "React/Cxx*/*.h"
+    ss.source_files         = "React/Cxx*/*.{h,m,mm}"
+  end
+
+  s.subspec "DevSupport" do |ss|
+    ss.dependency             "React/Core"
+    ss.dependency             "React/RCTWebSocket"
+    ss.source_files         = "React/DevSupport/*",
+                              "React/Inspector/*"
+  end
+
+  s.subspec "tvOS" do |ss|
+    ss.dependency             "React/Core"
+    ss.source_files         = "React/**/RCTTV*.{h, m}"
+  end
+
+  s.subspec "jschelpers" do |ss|
+    ss.dependency             "Folly", "2016.09.26.00"
```

Comment:
Do we need the version here for folly? If so, can we also pull it out from package.json

Reply:
removed react podspec